src/styles/blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  justify-content: space-between;
+  background-color: $white;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    font-size: 0;
```

Comment:
very confusing, you don't needed. If you use it for alignment try to use other means (e. g. paddings, flex alignment, etc.)